core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1040,11 +1040,13 @@ funds.withdrawal.inputs=Inputs selection
 funds.withdrawal.useAllInputs=Use all available inputs
 funds.withdrawal.useCustomInputs=Use custom inputs
 funds.withdrawal.receiverAmount=Receiver's amount
+funds.withdrawal.sendMax=Send Max
```

Comment:
Let's change this to "Send max available" for clarity, and update the other translations accordingly.